src/app/layout/pages/main-dashboard-page/main-dashboard-page.component.ts

```diff
@@ -20,10 +20,23 @@ export class MainDashboardPageComponent {
     this._store.select(AuthStateSelectors.isAuthenticated)
   )
 
-  constructor(private readonly _store: Store) {
+  private readonly _isLoggedOut = toSignal(
```

Comment:
when you name a variable with `is`, it implies the value of the variable will be boolean. This is not the case here. I will not get boolean (true/false) when I call `this._isLoggedOut()`, right?